Cargo.toml

```
@@ -6,30 +6,30 @@ edition = "2021"
 
 [dependencies]
 assert_matches = "1.5"
-async-trait = "0.1.53"
+async-trait = "0.1.68"
```

Let's remove the patch part to make it easier from all these deps - the patch version of the SemVer should always have backwards-compatible changes

And it will make crate version maintenance much easier.
```suggestion
async-trait = "0.1"
```

---

Good point, done :+1: